src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/VilkårUtil.kt

```diff
@@ -63,6 +63,7 @@ object VilkårUtil {
                 VilkårType.UTGIFTER_OVERNATTING -> false
                 VilkårType.LØPENDE_UTGIFTER_EN_BOLIG -> true
                 VilkårType.LØPENDE_UTGIFTER_TO_BOLIGER -> true
+                VilkårType.DAGLIG_REISE_OFFENTLIG_TRANSPORT -> false
```

Comment:
Ser ut som denne brukes for å slå sammen vilkår i oppsummerings kolonnen til venstre i løsningen. Lurer på om påfølgende daglig reise vilkår kan slåes sammen
```suggestion
                VilkårType.DAGLIG_REISE_OFFENTLIG_TRANSPORT -> true
```